packages/fast-components-class-name-contracts-base/package.json

```diff
@@ -5,12 +5,14 @@
   "version": "1.8.0",
   "author": {
     "name": "Microsoft",
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```

Comment:
Again, this is the default and readme key in package.json is not part of the [spec](https://docs.npmjs.com/files/package.json).